taqueria-vscode-extension/src/test/suite/extension.test.ts

```diff
@@ -5,102 +5,112 @@ import * as vscode from 'vscode';
+		// await workspace.updateWorkspaceFolders(0, 1, { uri: Uri.parse()});
 
-		// Run taqueria scaffold command
-		await vscode.commands.executeCommand('taqueria.scaffold');
-	});
+		setOpenDialogMocks(`${ligoContractFileDestination}`);
 
-	// TODO: https://github.com/ecadlabs/taqueria/issues/686
-	it.skip('Verify that VS Code command Taqueria Start Sandbox starts flextesa sandbox', async () => {
-		// Install plugin
-		await vscode.commands.executeCommand('taqueria.install @taqueria/plugin-flextesa');
+		await fse.rm(testProjectDestination, { recursive: true });
+		await fse.mkdir(testProjectDestination, { recursive: true });
 
-		// Verify that
-		await vscode.commands.executeCommand('taqueria.start_sandbox local');
-	});
-
-	// TODO: https://github.com/ecadlabs/taqueria/issues/686
-	it.skip('Verify that VS Code command Taqueria List Accounts display accounts on running sandbox', async () => {
-		await vscode.commands.executeCommand('taqueria.list_accounts local');
-	});
+		await vscodeMock.commands.executeCommand('taqueria.init');
 
-	// TODO: https://github.com/ecadlabs/taqueria/issues/686
-	it.skip('Verify that VS Code command Taqueria Stop Sandbox stops flextesa sandbox', async () => {
-		await vscode.commands.executeCommand('taqueria.stop_sandbox local');
-	});
+		choosePlugin = '@taqueria/plugin-ligo';
 
-	// TODO: https://github.com/ecadlabs/taqueria/issues/645
-	it.skip('Verify that VS Code command Taqueria Compile Ligo will compile Ligo contract', async () => {
-		// Install plugin
-		await vscode.commands.executeCommand('taqueria.install @taqueria/plugin-ligo');
+		await vscodeMock.commands.executeCommand('taqueria.install');
 
-		await vscode.commands.executeCommand('taqueria.compile_ligo');
+		choosePlugin = '@taqueria/plugin-flextesa';
 
-		// In last test or after all block
-		// vscode.commands.executeCommand("workbench.action.closeActiveEditor");
-	});
+		await vscodeMock.commands.executeCommand('taqueria.install');
 
-	// TODO: https://github.com/ecadlabs/taqueria/issues/687
-	it.skip('Verify that VS Code command Taqueria Compile SmartPy will compile SmartPy contract', async () => {
-		// Install plugin
-		await vscode.commands.executeCommand('taqueria.install @taqueria/plugin-smartpy');
+		// await vscodeMock.commands.executeCommand('taqueria.refresh_command_states');
 
-		await vscode.commands.executeCommand('taqueria.compile_smartpy');
-	});
+		// Copy contract from data folder
+		await fse.copyFile(ligoContractFileSource, ligoContractFileDestination);
 
-	// TODO: https://github.com/ecadlabs/taqueria/issues/688
-	it.skip('Verify that VS Code command Taqueria Deploy Contract will deploy contract to a test network', async () => {
-		// Install plugin
-		await vscode.commands.executeCommand('taqueria.install @taqueria/plugin-smartpy');
+		await vscodeMock.commands.executeCommand('taqueria.compile_pick_file');
 
-		await vscode.commands.executeCommand('taqueria.deploy');
+		assert.doesNotThrow(() => {
```

Comment:
If we are just testing that the expected file matches the file output I don't think we need to wrap this all in an assertion that no error happens. If an error occurs the assertion on the files should tell us what happened

Reply:
@alexzbusko No, we are also testing that file has been created. If it is not created it throws an error. That is why it is wrapped into doesNotThrow assertion here.